src/components/__tests__/rangeSelect.test.tsx

```diff
@@ -0,0 +1,59 @@
+import React from "react"
+import { fireEvent, render } from "@testing-library/react"
+
+import RangeSelect from "../rangeSelect"
+
+const onChangeMin = jest.fn()
+const onChangeMax = jest.fn()
+
+const renderWrapper = ({
+  name = "name",
+  minValue = null,
+  maxValue = null,
+  minPlaceholder = "min",
+  maxPlaceholder = "max",
+  handleChangeMin = onChangeMin,
+  handleChangeMax = onChangeMax,
+  options = [
+    { name: "1", value: 1 },
+    { name: "2", value: 2 },
+    { name: "3", value: 3 },
+    { name: "4", value: 4 },
+    { name: "5", value: 5 },
+    { name: "6", value: 6 },
+  ],
+} = {}) =>
+  render(
+    <RangeSelect
+      name={name}
+      value={{
+        min: minValue,
+        max: maxValue,
+      }}
+      placeholder={{
+        min: minPlaceholder,
+        max: maxPlaceholder,
+      }}
+      handleChange={{
+        min: handleChangeMin,
+        max: handleChangeMax,
+      }}
+      options={options}
+    />
+  )
+
+describe("RangeSelect", () => {
+  it("renders RangeSelect correctly", () => {
+    const { container } = renderWrapper()
+    expect(container).toMatchSnapshot()
+  })
```

Comment:
I'm not sure this test brings a lot of value. The resulting markup is large and the slightest change will result in this test failing. How about checking that there are two selects with expected names instead?